hadoop-hdds/server-scm/src/main/java/org/apache/hadoop/hdds/scm/node/NodeManager.java

```diff
@@ -342,6 +342,24 @@ int getNodeQueuedCommandCount(DatanodeDetails datanodeDetails,
    */
   List<DatanodeDetails> getNodesByAddress(String address);
```

Comment:
Why is address not sufficient to cover both `hostname` as well as `ip`

Reply:
This method uses a map that either has the `ip` addresses or the `hostnames`. We would like to have both of them available in the CLI at all times.